apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-settings/vo-settings-sponsored-members/vo-settings-sponsored-members.component.ts

```diff
@@ -21,57 +21,57 @@ import { SponsorExistingMemberDialogComponent } from '../../../../../shared/comp
 import { Urns } from '@perun-web-apps/perun/urns';
 import { Role } from '@perun-web-apps/perun/models';
 
+interface AuthPrivilege {
+  readAuth: boolean;
+  manageAuth: boolean;
+  modes: string[];
+}
+
```

Comment:
Later on we should unify definition of this interface, since it is needed also in `perun-utils`.